templates/layouts/aw-install-simg.hbs

```diff
@@ -15,58 +15,85 @@
       <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
       {{> body }}
       <h1>Preparation</h1>
-      <div class="install-preparation-box">
-          <img src="{{assets}}/img/{{deviceName}}.jpg" class="install-preparation-img"><br>
-      </div>
       <div class="install-preparation-box">
         <h3>Download AsteroidOS 1.0</h3>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/asteroid-image-{{deviceName}}.ext4" role="button">asteroid-image-{{deviceName}}.ext4</a>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/zImage-dtb-{{deviceName}}.fastboot" role="button">zImage-dtb-{{deviceName}}.fastboot</a>
-        <h3>Install Fastboot & Adb</h3>
+        <br><br><i>Place the files in your users "Downloads" folder for the later commands to work</i><br>
+        <h3>Install Fastboot & ADB</h3>
+        <b>Linux systems</b> with apt package manager
+        <br><br>
         <pre><code class="bash">sudo apt install android-tools-adb android-tools-fastboot android-tools-fsutils</code></pre>
-        <p><i>Note: previous command may differ depending on your OS</i></p>
+        <br>
+        <b>Windows users</b> download this installer <a href="https://forum.xda-developers.com/showthread.php?t=2317790">published on XDA by shimp208</a>
```

Comment:
I'm not super crazy about the idea of asking our users to run on their computer a program compiled by a complete stranger. This could be ok for now but we might want to find a workaround and better sooner than later.

Reply:
looking for an alternative.
The solution is using adb 1.0.39. simplz. No real need for an installer i guess.